### PR TITLE
Test out-of-tree projects

### PR DESCRIPTION
**Pull request by privat**
This is a simple script that allows jenkins (and human users) to test out-of-tree projects that live in other repositories.

The main file is `contrib/oot.txt` that contains the various projects and `contrib/oot.sh` that tries to build them.

A preview of the current state is available at:
http://gresil.org/jenkins/job/nit_oot/lastCompletedBuild/testReport/

I will improve the script to add checks.

### CODE REVIEW

File: contrib/oot.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Script to update and check out-of-tree projects from oot.txt
```
Comment (Morriar) [inline contrib/oot.sh]: Why does this file have a different header than the other?

Comment (privat) [inline contrib/oot.sh]: you are right

**Comment by privat** [comment]
test oot please

**Comment by privat** [comment]
test oot please

**Pull request merged**